.gitignore

```diff
@@ -1,5 +1,7 @@
 # Local .terraform directories
 **/.terraform/*
+#Local node_module directories from make install
+**/.node_modules/*
 
```

Comment:
`node_modules/` would also do the job. Side note: I don't thinl `/.node_modules/` will work since the directory is `node_modules/` not `.node_modules/`.

You should probably change this.

Reply:
terraform and node_modules pathing updated in .gitignore, changes commited.